src/wavm/ThreadState.cpp

```diff
@@ -0,0 +1,120 @@
+#include <wavm/ThreadState.h>
+#include <wavm/WAVMWasmModule.h>
```

Comment:
As with the `ThreadState.h` file, this is a single place for all the implementation parts we'll be keeping